src/tests/ftest/pool/pda.py

```diff
@@ -17,18 +16,22 @@ class PoolPDAProperty(TestWithServers):
 
     :avocado: recursive
     """
-    def verify_cont_pda(self, expected_value, match_str):
+    def verify_cont_pda(self, expected_value, match_name):
         """
         Verify the container pda property.
 
         Args:
             expected_value (int): expected container pda value
+            match_name (str): name field used to obtain the value from the output
         """
-        daos_cmd = self.get_daos_command()
-        cont_prop = daos_cmd.container_get_prop(self.pool.uuid,
-                                                self.container.uuid)
-        cont_prop_stdout = cont_prop.stdout_text
-        pda_value = int(re.search(match_str + r".*([\d]+)", cont_prop_stdout).group(1))
+        pda_value = None
+
+        cont_props = self.container.get_prop()
```

Comment:
Just wondering, could we make an optional parameter to get_prop() something like specified_property=None, if this is specified return this property back, otherwise, return all properties.

Reply:
Done. Thanks for pointing out.